### PR TITLE
#18345 on editmode we need to set the page mode to preview

### PR DESCRIPTION
**Pull request by jdotcms**
On edit mode we need to set the page mode to preview in order to display images on dA servlet that haven't live version

### CODE REVIEW

**Review by erickgonzalez — APPROVED** [review]

File: dotCMS/src/main/webapp/html/portlet/ext/contentlet/edit_contentlet.jsp [hunk 1]
```diff
@@ -195,6 +195,8 @@
 	}
 
 	boolean isLocked=(request.getParameter("sibbling") != null) ? false : contentlet.isLocked();
+
+	PageMode.setPageMode(request, PageMode.PREVIEW_MODE);
```
Comment (freddyucv) [inline dotCMS/src/main/webapp/html/portlet/ext/contentlet/edit_contentlet.jsp:199]: I think this change is dangerous

**Review by freddyucv — CHANGES_REQUESTED** [review]

**Comment by fmontes** [comment]
Needs another approach

**Comment by wezell** [comment]
PR: #18561

**Comment by fmontes** [comment]
Chat with Freddy he mention what to test.

**Pull request merged**